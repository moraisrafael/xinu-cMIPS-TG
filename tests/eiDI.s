	.include "cMIPS.s"
	.text
	.align 2
	.globl _start
	.ent _start
_start: li  $5, 3
	la  $15, x_IO_BASE_ADDR
	nop
lasso:	ei   $8
	sw   $8, 0($15)  # display old STATUS's value
	nop
	di   $8
	sw   $8, 4($15)  # display old STATUS's value
	addiu $5,$5,-1
	bne $5,$0,lasso
	nop
        nop
	nop
        wait
        nop
	nop
	nop
	.end _start
