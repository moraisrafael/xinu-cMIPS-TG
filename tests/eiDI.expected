10000002
10000003
10000002
10000003
10000002
10000003
